Dockerfile

```diff
@@ -0,0 +1,19 @@
+# base image
+FROM mariadb:latest
+
+# set the working directory in the container
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+ENV PYTHONPATH /
+
+# get necessary python libraries
+COPY requirements.txt .
```

Comment:
Are the python libraries actually required here? I had a similar question on Ed's too. I don't think I had to use any for mine.